File: Algorithms in Golang/linear-search.go
// Linear Search
// Based on .cpp reference
// Array size is 10

package main

import (
	"fmt"
)

// Linear Search Function
func linearSearch(list [10]int, find int){
	for i := 0; i < 10; i++ {
		if list[i] == find {
			fmt.Println("Element found at position ", i+1)
			return
		}
  }

		fmt.Println("Element not found")
		return

}

// Main function
func main(){
	var list [10]int
	var find int

  fmt.Println("Enter an element : ")

	for i := 0; i < 10; i++ {
		fmt.Scan(&list[i])
	}

	fmt.Println("Find an element : ")
	fmt.Scan(&find)
	linearSearch(list, find)

}

File: Algorithms in C#/linear-search.cs
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System;

namespace LinearSearch {
    class Program {
        public static void Main (string[] args) {
            int search_item;
            int pos = 0;

            List<int> list = new List<int>();

            Console.Write("How many numbers you want to enter? ");
            int n = int.Parse(Console.ReadLine());
            Console.WriteLine("Enter the numbers\n");
            for (int i = 0; i < n; i++)
            {
                Console.Write("Number " + (i+1) + ": ");
                list.Add(int.Parse(Console.ReadLine()));
            }

            Console.Write ("Enter item to search : ");
            search_item = int.Parse (Console.ReadLine ());

            //Loop to search element in array.
            for (int i = 0; i < list.Count; i++) {
                if (search_item == list[i]) {
                    pos = i + 1;
                    break;
                }
            }

            if (pos == 0) {
                Console.WriteLine ("Item Not found in array");
            } else {
                Console.WriteLine ("Position of item in array: " + pos);
            }

        }
    }
}